flavors/README.md

```diff
@@ -40,35 +40,38 @@ The template-Exasol-all-python-3.10-conda flavor provides the language Python 3.
 
 - template-Exasol-all-python-3.10-conda
   - Base Image: ubuntu:22.04
-  - Compatible Exasol Versions: 8.0.\*,7.1.\*, 7.0.\*
+  - Compatible Exasol Versions: 8.\*,7.1.\*, 7.0.\*
   - Available languages: Python 3.10
   - Supported Package Manager: conda, pip
   - [Here](template-Exasol-all-python-3.10-conda/FLAVOR_DESCRIPTION.md) you can find more details about the flavor, such as the provided packages.
 
 ### template-Exasol-8-python-3.10-cuda-conda
 
-The template-Exasol-8-python-3.10-cuda-conda flavor provides the language Python 3.10 with NVIDIA GPU support and with only absolute necessary packages. This allows very simple customization because you control almost all dependencies. In contrast to template-Exasol-all-python-3.10, it also allows the installation of conda packages.
+The template-Exasol-8-python-3.10-cuda-conda flavor provides the language Python 3.10 with NVIDIA GPU support and with only absolute necessary packages. The flavor contains the Cuda-SDK and is prepared to work with the Nvidia-Drivers, which however need to be provided by the Host OS. This allows very simple customization because you control almost all dependencies. In contrast to template-Exasol-all-python-3.10, it also allows the installation of conda packages.
```

Comment:
```suggestion
The template-Exasol-8-python-3.10-cuda-conda flavor provides the language Python 3.10 with NVIDIA GPU support and with only absolute necessary packages. The flavor contains the Cuda-SDK and is prepared to work with the Nvidia-Driver, which however needs to be provided by the Host OS. This allows very simple customization because you control almost all dependencies. In contrast to template-Exasol-all-python-3.10, it also allows the installation of conda packages.
```